### ISSUE TITLE
Bad reconstruction result on my custom dataset (Only the head was rebuilt)

### ISSUE THREAD

**Issue by chuxiang93** [autonomousvision/sdfstudio#237]
**Describe the bug**
Bad reconstruction result on my custom dataset. The reconstructed target is less than half the size.

**2d image:**
![000312_rgb](https://github.com/autonomousvision/sdfstudio/assets/11666931/12b0736b-b6d0-499f-996c-ee766f4b5c5b)
![000257_rgb](https://github.com/autonomousvision/sdfstudio/assets/11666931/6d69294c-1e23-4b89-98e6-300886c009c6)
![000214_rgb](https://github.com/autonomousvision/sdfstudio/assets/11666931/28300462-436c-45eb-84d9-748261c09b78)
**mesh result:**
<img width="676" alt="image" src="https://github.com/autonomousvision/sdfstudio/assets/11666931/f501641c-6f1f-4770-93ef-19d60ef2c9e2">

**To Reproduce**
1，dataset conversion
`python scripts/datasets/process_nerfstudio_to_sdfstudio.py --data-type "colmap" --scene-type "object" --data data/nerfstudio/toy  --output-dir data/sdfstudio/toy --mono-prior --omnidata-path ../workspace/omnidata/omnidata_tools/torch/ --pretrained-models ../workspace/omnidata/omnidata_tools/torch/pretrained_models/`

2，train code
`ns-train neus-facto --pipeline.model.sdf-field.inside-outside False --pipeline.model.sdf-field.bias 0.3 --pipeline.model.background-model mlp --pipeline.model.mono-depth-loss-mult 0.1 --pipeline.model.mono-normal-loss-mult 0.05 --experiment-name neus-facto-toy sdfstudio-data --data data/sdfstudio/toy --include_mono_prior True`

**Comment by z-x-j** [comment]
Have you resolved this issue?

**Comment by a1445605672** [comment]
I also encountered this problem, how did you solve it?
